tests/test_merging.py

```diff
@@ -0,0 +1,23 @@
+"""Test JSON schema merging."""
+import glob
+import json
+from pathlib import Path
+
+import pytest
+
+from json_schema_fuzz.merging import merge
+
+THIS_DIR = Path(__file__).parent
+CASE_DIR = THIS_DIR / "merge_cases"
+case_files = glob.glob(str(CASE_DIR / "*.json"))
+cases = []
+for filename in case_files:
```

Comment:
I prefer the json-in-code structure that we use in strider rather than the separated style. I think it's cleaner to have everything in one file rather than a bunch of files in the repository.

Reply:
Especially if we are going to be throwing validation errors, it's going to be easier to have the data in the code

Reply:
For error checks, we'll have to add a separate test function.